source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/ChoiceSetInputRenderer.java

```diff
@@ -246,6 +248,8 @@ public View renderComboBox(
 
         final ComboBoxInputHandler comboBoxInputHandler = new ComboBoxInputHandler(choiceSetInput);
         final Spinner spinner = new Spinner(context);
+        spinner.getBackground().setColorFilter(Color.RED, PorterDuff.Mode.SRC_ATOP);
```

Comment:
>spinner.getBackground().setColorFilter(Color.RED, PorterDuff.Mode.SRC_ATOP); [](start = 8, length = 76)

For my own curiosity, what does this line do?

Reply:
that was a line that should not be there